### PR TITLE
Mention upload in README

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,15 @@ Delete a setting [with](https://s3.amazonaws.com/foliodocs/api/mod-settings/sett
 
 This is a write operation and returns 204 if the setting was deleted.
 
+It's also possible to upload (or "import") settings
+[with](https://s3.amazonaws.com/foliodocs/api/mod-settings/settings.html#operation/uploadSettings):
+
+    PUT /settings/upload
+
+Settings are created/updated with this service. The provided settings must
+not include an identifier. An identifier will be assigned by the server when
```
Comment (MikeTaylor) [inline README.md:109]: So settings are identified by scope/key/user combination? That's surprising to me, I'd have thought literally the key use of an `id` field is to, well, identify the record for such operations.

Comment (adamdickmeiss) [reply README.md:109]: key, scope must be unique (or with userId if present too).

Comment (MikeTaylor) [reply README.md:109]: I should have been clearer, sorry. I was asking for this information to be made explicit in the README.

Comment (adamdickmeiss) [reply README.md:109]: Sure. That should go into the description of the elements ealier, then.

**Review by MikeTaylor — APPROVED** [review]
LGTM

**Pull request merged**